src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+@import url(./stars.scss);
+@use "./variables" as v;
+
+$main-color: rgb(6, 11, 53);
+$secoundary-color: rgb(97, 96, 112);
+$distance-to-border: 16px;
+
+.card {
+  font-family: "Roboto", sans-serif;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px $distance-to-border 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    padding: 4px 0 0;
+    color: $secoundary-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 22px;
+  }
+
+  &__reviews-info {
+    font-size: 10px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $secoundary-color;
+  }
+
+  &__money-amount {
+    font-size: 16px;
+    line-height: 17px;
+    font-weight: 700;
+  }
+
+  &__button {
+    padding: 12px 69px;
```

Comment:
try to add width: 100% instead side padding, becuse this button should be full width of card